base_ubl/models/ubl.py

```diff
@@ -633,11 +633,15 @@ def ubl_parse_party(self, party_node, ns):
         website_xpath = party_node.xpath("cbc:WebsiteURI", namespaces=ns)
         partner_dict = {
             "vat": vat_xpath and vat_xpath[0].text or False,
-            "name": partner_name_xpath[0].text,
+            "name": partner_name_xpath and partner_name_xpath[0].text or False,
             "email": email_xpath and email_xpath[0].text or False,
             "website": website_xpath and website_xpath[0].text or False,
             "phone": phone_xpath and phone_xpath[0].text or False,
         }
+        id_node.xpath("cac:PartyIdentification/cbc:ID", namespaces=ns)
+        if id_node:
+            partner_dict["id"] = id_node[0].text
```

Comment:
As far as I understand this `"id"` should be the identification document number? Could `"id"` here be confused with the id of the record? In this case better to use another string.

Reply:
renamed to id_number as in https://github.com/OCA/partner-contact/blob/13.0/partner_identification/models/res_partner_id_number.py